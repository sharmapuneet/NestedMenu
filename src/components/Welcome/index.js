import React from 'react'
import Typography from '@material-ui/core/Typography'

const Welcome = () => {
  return (
    <>
      <Typography variant="h4">
        Welcome to Nested Menu.
      </Typography>
      <Typography variant="subtitle1">
        This project is made to show how we can make Nested menu with unlimited nested menu items. 
      </Typography>
    </>
  )
}

export default Welcome
