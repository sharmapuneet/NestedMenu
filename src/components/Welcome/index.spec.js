import React from 'react'
import { shallow, render } from 'enzyme'
import Welcome from './index'

describe('Welcome', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Welcome />)

    expect(wrapper).toMatchSnapshot()
  })

  it('should have text', () => {
    const wrapper = render(<Welcome />)

    expect(wrapper.text()).toBe('Welcome to Nested Menu.This project is made to show how we can make Nested menu with unlimited nested menu items.')
  })
})
