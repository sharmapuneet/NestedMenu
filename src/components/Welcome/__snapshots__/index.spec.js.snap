// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should render correctly 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Typography))
    variant="h4"
  >
    Welcome to Nested Menu.
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Typography))
    variant="subtitle1"
  >
    This project is made to show how we can make Nested menu with unlimited nested menu items.
  </WithStyles(ForwardRef(Typography))>
</Fragment>
`;
